hedera-node/hedera-app/src/xtest/java/token/TokenUpdateXTest.java

```diff
@@ -0,0 +1,151 @@
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .supplyKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .feeScheduleKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .pauseKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .metadataKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .adminKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+    }
+
+    @Override
+    protected Map<TokenID, Token> initialTokens() {
+        /**
+         * Initial token key must always have initialized all keys. IMMUTABILITY_SENTINEL_KEY is the default value for keys.
```

Comment:
`IMMUTABILITY_SENTINEL_KEY` is not the default value for keys, it is the key value to consider the key as "_removed_"
could be changed to "Initial token must always have initialized all keys. Set DEFAULT_KEY as default value for all keys."

Reply:
Updated, thanks!